website/docs/r/service_dictionary_items_v1.html.markdown

```diff
@@ -0,0 +1,212 @@
+
+  domain {
+    name    = "demo.notexample.com"
+    comment = "demo"
+  }
+
+  backend {
+    address = "demo.notexample.com.s3-website-us-west-2.amazonaws.com"
+    name    = "AWS S3 hosting"
+    port    = 80
+  }
+
+  dictionary {
+    name = var.mydict.name
+  }
+
+  force_destroy = true
+  }
+
+resource "fastly_service_dictionary_items_v1" "items" {
+  service_id = fastly_service_v1.myservice.id
+  dictionary_id = {for d in fastly_service_v1.myservice.dictionary : d.name => d.dictionary_id}[var.mydict.name]
+  items = var.mydict.items
+}
+
+```
+
+Expression and functions usage:
+
+```hcl
+// Local variables used when formating values for the "My Project Dictionary" example
+locals {
+  dictionary_name = "My Project Dictionary"
+  host_base = "demo.ocnotexample.com"
+  host_divisions = ["alpha", "beta", "gamma", "delta"]
+}
+
+// Define the standard service that will be used to manage the dictionaries.
+resource "fastly_service_v1" "myservice" {
+  name = "demofastly"
+
+  domain {
+    name = "demo.ocnotexample.com"
+    comment = "demo"
+  }
+
+  backend {
+    address = "demo.ocnotexample.com.s3-website-us-west-2.amazonaws.com"
+    name = "AWS S3 hosting"
+    port = 80
+  }
+
+  dictionary {
+    name = local.dictionary_name
+  }
+
+  force_destroy = true
+}
+
+// This resource is dynamically creating the items from the local variables through for expressions and functions.
+resource "fastly_service_dictionary_items_v1" "project" {
+  service_id = fastly_service_v1.myservice.id
+  dictionary_id = {for d in fastly_service_v1.myservice.dictionary : d.name => d.dictionary_id}[local.dictionary_name]
+  items = {
+    for division in local.host_divisions:
+      division => format("%s.%s", division, local.host_base)
+  }
+
+}
+```
+
+Lifecyle ignore_changes usage:
+
+The following example demonstrates how the lifecycle ignore_change field can be used to suppress updates against the 
+items in a dictionary.  If an external means, (for example via the Fastly API or UI) is used to manage items in a dictionary, then this will stop Terraform realigning the remote state.
+
+```hcl
+...
+
+resource "fastly_service_dictionary_items_v1" "items" {
+    service_id = "${fastly_service_v1.myservice.id}"
+    dictionary_id = "${{for s in fastly_service_v1.myservice.dictionary : s.name => s.dictionary_id}[var.mydict_name]}"
+    items = {
+        key1: "value1"
+        key2: "value2"
+    }
+    
+    lifecycle {
+      ignore_changes = [items,]
+    }
+}
+
+
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `service_id` - (Required) The ID of the Service that the dictionary belongs to
+* `dictionary_id` - (Required) The ID of the dictionary that the items belong to
+* `items` - (Optional) A Map representing an entry in the dictionary, (key/value)
+
+
+## Attributes Reference
+
+[fastly-dictionary]: https://docs.fastly.com/api/config#dictionary
+[fastly-dictionary_item]: https://docs.fastly.com/api/config#dictionary_item
```

Comment:
Is this valid markdown? 🤔

Reply:
This was correct.  The markdown was not correct.  The links have been updated and now display.